ReSharper.FSharp/test/data/parsing/XmlDoc 03 - Empty.fs

```diff
@@ -0,0 +1,2 @@
+/// 
+let x = ()
```

Comment:
Would it make sense to also cover cases like these?

A space after the slashes:
```fsharp
/// 
let x = ()
```

Multiple empty lines:
```fsharp
///
///
let x = ()
```

Sorry if it's already covered 🙂

Reply:
The first one case is already covered, others can be added, yes